tests/run-make/linker-flags-from-attribute-from-dep/Makefile

```diff
@@ -0,0 +1,10 @@
+include ../tools.mk
+
+all:
+	# Build deps
+	$(RUSTC) native_dep_1.rs --crate-type=staticlib
+	$(RUSTC) native_dep_2.rs --crate-type=staticlib
+	$(RUSTC) rust_dep.rs --crate-type=lib
+
+	# Check sequence of linker args
+	$(RUSTC) main.rs --extern lib=$(TMPDIR)/librust_dep.rlib --crate-type=bin --print link-args | $(CGREP) -e 'native_dep_1.*some_flag.*native_dep_2'
```

Comment:
This is an adaptation of `tests\run-make\pass-linker-flags-from-dep\Makefile`, right?
Maybe it's better to just merge it into that test to keep the similar testing logic in the same makefile.

Same for `tests/run-make/linker-flags-from-attribute(-flavor)/Makefile`, which are adaptations of other tests.